Python/optimizer_bytecodes.c

```diff
@@ -846,6 +846,17 @@ dummy_func(void) {
        next = sym_new_type(ctx, &PyLong_Type);
     }
 
+    op(_CALL_TYPE_1, (callable, null, arg -- res)) {
+        (void)callable;
+        (void)null;
```

Comment:
This should work!

```suggestion
    op(_CALL_TYPE_1, (unused, unused, arg -- res)) {
```

Reply:
Hmm that doesn't help silence the warnings in `executor_cases.c.h` :/ only thing that I found that works is `(void)foo` like in [0925179](https://github.com/python/cpython/pull/132419/commits/09251797a9f41fd97cec71e2782f59b7c25e80be). Is there a better way?

Reply:
Nope, that's fine!